### ISSUE TITLE
Inconsistent names when using language with diacritics

### ISSUE THREAD

**Issue by nzerr** [wassim-k/sp-entity#3]
When performing the REST api requests, the response returns the default language that is configured on the SharePoint site.
In some languages, diacritics prevent to generate a correct name for the entity fields. 

For instance, on a site French, "Créé" results in "cr", "Modifié Par" results in "modifiPar" etc.

I think there are 2 options :

- Either use the internal name of the field to generate the entities

- Or remove the diacritics, here is a suggestion on how to do it (in TextUtil.ts)

```typescript
public static camelCase(text: string): string {

        text = this.removeDiacritics(text);
        text = this.removeNonWordChars(text);
        return camelCase(text);
    }

    public static removeDiacritics(text: string): string {

        return text.normalize('NFD').replace(/[\u0300-\u036f]/g, '');
    }
```

**Comment by wassim-k** [comment]
Tanks @nzerr for the detailed description and the code suggestion.

I don't think we can trust internal field names to be always useful, so I prefer the second solution to remove diacritics instead.

Are you able to create a PR and test the solution? I currently don't have a SharePoint environmnet at may need some time to set one up to test this.

**Comment by wassim-k** [comment]
This has been implemented in v1.2.0
